src/postgres/commands/createPostgresServer/steps/PostgresServerSkuStep.ts

```diff
@@ -27,17 +25,28 @@ export class PostgresServerSkuStep extends AzureWizardPromptStep<IPostgresServer
             "selectPostgresSku",
             "Select the Postgres SKU and options."
         );
-        wizardContext.sku = (
-            await ext.ui.showQuickPick(this.getPicks(), { placeHolder })
-        ).data;
+        const pricingTiers: IAzureQuickPickItem<Sku | undefined>[] = await this.getPicks();
+        const moreInfo: IAzureQuickPickItem = { label: localize('ShowPricingCalculator', '$(link-external) Show pricing information...'), data: undefined }
```

Comment:
You can use `onPicked` like [here](https://github.com/microsoft/vscode-azuretools/blob/cc46259643436649b85f40cc9e843d1ef95d19a7/ui/src/wizard/StorageAccountListStep.ts#L148) to simplify this code. It should get rid of the need for a loop and also doesn't close & reopen the quick pick when the url is clicked